aesara/scan/op.py

```diff
@@ -3440,7 +3440,7 @@ def profile_printer(
             )
 
 
-@op_debug_information.register(Scan)
+@op_debug_information.register(Scan)  # type: ignore[has-type]
```

Comment:
This looks like it's in response to https://github.com/aesara-devs/aesara/issues/1221.